<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>adminWidget</class>
 <widget class="QFrame" name="adminWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>633</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CookBook | Admin mode</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>641</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(221, 221, 221);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="h1_adm">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>10</y>
      <width>611</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Source Code Pro</family>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>CookBook_Admin</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>441</width>
     <height>331</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(221, 221, 221);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Source Code Pro</family>
     </font>
    </property>
    <property name="text">
     <string>Edit Tables</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Source Code Pro</family>
     </font>
    </property>
    <property name="text">
     <string>Command prompt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>170</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Source Code Pro</family>
     </font>
    </property>
    <property name="text">
     <string>Statistics</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exit_adm_btn">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>240</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Source Code Pro</family>
     </font>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QFrame" name="edit_tables_frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>441</width>
      <height>331</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="meal_table_btn">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>10</y>
       <width>321</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Edit &quot;meal&quot;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="category_btn">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>90</y>
       <width>321</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Edit &quot;category&quot;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>170</y>
       <width>321</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Edit &quot;ingredient&quot;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>250</y>
       <width>321</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
    <widget class="QFrame" name="meal_edit_frame">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>451</width>
       <height>341</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QTableView" name="meal_table_view">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>421</width>
        <height>181</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Source Code Pro</family>
       </font>
      </property>
     </widget>
     <widget class="QLineEdit" name="meal_name">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>141</width>
        <height>27</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Source Code Pro</family>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>meal_name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="meal_desc">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>200</y>
        <width>141</width>
        <height>27</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Source Code Pro</family>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>meal_desc</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="meal_htc">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>200</y>
        <width>141</width>
        <height>27</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Source Code Pro</family>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>meal_htc</string>
      </property>
     </widget>
     <widget class="QPushButton" name="insert_meal">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>280</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Source Code Pro</family>
       </font>
      </property>
      <property name="text">
       <string>Insert meal</string>
      </property>
     </widget>
     <widget class="QPushButton" name="update_meal">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>230</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Source Code Pro</family>
       </font>
      </property>
      <property name="text">
       <string>Update meal</string>
      </property>
     </widget>
     <widget class="QPushButton" name="delete_meal">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>280</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Source Code Pro</family>
       </font>
      </property>
      <property name="text">
       <string>Delete meal</string>
      </property>
     </widget>
     <widget class="QPushButton" name="back_to_select_table">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>230</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Source Code Pro</family>
       </font>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="meal_id">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>260</y>
        <width>141</width>
        <height>27</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Source Code Pro</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>meal_id</string>
      </property>
     </widget>
     <widget class="QFrame" name="meal_edit_frame_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>451</width>
        <height>341</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QTableView" name="meal_table_view_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>421</width>
         <height>181</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Source Code Pro</family>
        </font>
       </property>
      </widget>
      <widget class="QLineEdit" name="category_name">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>200</y>
         <width>151</width>
         <height>27</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Source Code Pro</family>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>category_name</string>
       </property>
      </widget>
      <widget class="QPushButton" name="insert_category">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>280</y>
         <width>151</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Source Code Pro</family>
        </font>
       </property>
       <property name="text">
        <string>Insert category</string>
       </property>
      </widget>
      <widget class="QPushButton" name="update_category">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>230</y>
         <width>151</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Source Code Pro</family>
        </font>
       </property>
       <property name="text">
        <string>Update category</string>
       </property>
      </widget>
      <widget class="QPushButton" name="delete_category">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>280</y>
         <width>161</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Source Code Pro</family>
        </font>
       </property>
       <property name="text">
        <string>Delete category</string>
       </property>
      </widget>
      <widget class="QPushButton" name="back_to_select_table_2">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>230</y>
         <width>161</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Source Code Pro</family>
        </font>
       </property>
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="category_id">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>200</y>
         <width>141</width>
         <height>27</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Source Code Pro</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>category_id</string>
       </property>
      </widget>
      <widget class="QFrame" name="meal_edit_frame_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>451</width>
         <height>341</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QTableView" name="meal_table_view_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>421</width>
          <height>181</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Source Code Pro</family>
         </font>
        </property>
       </widget>
       <widget class="QLineEdit" name="ingredient_name">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>200</y>
          <width>151</width>
          <height>27</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Source Code Pro</family>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="placeholderText">
         <string>ingredient_name</string>
        </property>
       </widget>
       <widget class="QPushButton" name="insert_ingredient">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>300</y>
          <width>181</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Source Code Pro</family>
         </font>
        </property>
        <property name="text">
         <string>Insert ingredient</string>
        </property>
       </widget>
       <widget class="QPushButton" name="update_ingredient">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>270</y>
          <width>181</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Source Code Pro</family>
         </font>
        </property>
        <property name="text">
         <string>Update ingredient</string>
        </property>
       </widget>
       <widget class="QPushButton" name="delete_ingredient">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>300</y>
          <width>191</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Source Code Pro</family>
         </font>
        </property>
        <property name="text">
         <string>Delete ingredient</string>
        </property>
       </widget>
       <widget class="QPushButton" name="back_to_select_table_3">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>270</y>
          <width>191</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Source Code Pro</family>
         </font>
        </property>
        <property name="text">
         <string>Back</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="ingredient_id">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>200</y>
          <width>141</width>
          <height>27</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Source Code Pro</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="placeholderText">
         <string>ingredient_id</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="ingredient_cost_unit">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>200</y>
          <width>131</width>
          <height>27</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Source Code Pro</family>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="placeholderText">
         <string>ingredient_cu</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="ingredient_nutval">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>230</y>
          <width>151</width>
          <height>27</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Source Code Pro</family>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="placeholderText">
         <string>ingredient_nutv</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="cat_ingred">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>230</y>
          <width>151</width>
          <height>27</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Source Code Pro</family>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="placeholderText">
         <string>cat_ingredient</string>
        </property>
       </widget>
       <widget class="QFrame" name="frame_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>441</width>
          <height>331</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLabel" name="cp_label">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>10</y>
           <width>201</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Source Code Pro</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Command prompt</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLineEdit" name="cp_le">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>60</y>
           <width>351</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Source Code Pro</family>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QTableView" name="tv_for_cp">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>130</y>
           <width>351</width>
           <height>181</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
        </widget>
        <widget class="QPushButton" name="execute_btn">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>100</y>
           <width>171</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Execute</string>
         </property>
        </widget>
        <widget class="QPushButton" name="back_to_menu">
         <property name="geometry">
          <rect>
           <x>230</x>
           <y>100</y>
           <width>171</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Back</string>
         </property>
        </widget>
        <widget class="QFrame" name="stat_frame">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>441</width>
           <height>331</height>
          </rect>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLabel" name="stat_label">
          <property name="geometry">
           <rect>
            <x>70</x>
            <y>10</y>
            <width>311</width>
            <height>20</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Source Code Pro</family>
           </font>
          </property>
          <property name="text">
           <string>Statistics</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QTableView" name="tv_for_mtv_1">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>40</y>
            <width>201</width>
            <height>131</height>
           </rect>
          </property>
         </widget>
         <widget class="QTableView" name="tv_for_mtv_2">
          <property name="geometry">
           <rect>
            <x>230</x>
            <y>40</y>
            <width>201</width>
            <height>131</height>
           </rect>
          </property>
         </widget>
         <widget class="QTableView" name="tv_for_mtv_3">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>180</y>
            <width>201</width>
            <height>131</height>
           </rect>
          </property>
         </widget>
         <widget class="QTableView" name="tv_for_mtv_4">
          <property name="geometry">
           <rect>
            <x>230</x>
            <y>180</y>
            <width>201</width>
            <height>131</height>
           </rect>
          </property>
         </widget>
         <widget class="QPushButton" name="exit_stat_button">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>6</y>
            <width>88</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>EXIT</string>
          </property>
         </widget>
        </widget>
       </widget>
       <zorder>back_to_select_table_3</zorder>
       <zorder>meal_table_view_3</zorder>
       <zorder>ingredient_name</zorder>
       <zorder>insert_ingredient</zorder>
       <zorder>update_ingredient</zorder>
       <zorder>delete_ingredient</zorder>
       <zorder>ingredient_id</zorder>
       <zorder>ingredient_cost_unit</zorder>
       <zorder>ingredient_nutval</zorder>
       <zorder>cat_ingred</zorder>
       <zorder>frame_3</zorder>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
